electron-app/package.json

```diff
@@ -62,13 +86,137 @@
+    "detectUpdateChannel": false,
+    "generateUpdatesFilesForAllChannels": true,
+    "npmRebuild": false,
+    "directories": {
+      "buildResources": "resources"
+    },
+    "fileAssociations": [
+      {
+        "ext": "ino",
+        "role": "Editor"
+      }
+    ],
+    "files": [
+      "arduino-ide-electron-main.js",
+      "src-gen",
+      "lib",
+      "!**node_modules/**"
+    ],
+    "extraResources": [
+      {
+        "from": "plugins",
+        "to": "app/plugins"
+      }
+    ],
+    "win": {
+      "target": [
+        "msi",
+        "nsis",
+        "zip"
+      ]
+    },
+    "mac": {
+      "darkModeSupport": true,
+      "hardenedRuntime": true,
+      "gatekeeperAssess": false,
+      "entitlements": "resources/entitlements.mac.plist",
+      "entitlementsInherit": "resources/entitlements.mac.plist",
+      "target": {
+        "target": "default"
+      }
+    },
+    "linux": {
+      "target": [
+        "zip",
+        "AppImage"
+      ],
+      "category": "Development",
+      "icon": "resources/icons"
+    },
+    "msi": {
+      "runAfterFinish": false
+    },
+    "nsis": {
+      "oneClick": false,
+      "installerHeaderIcon": "resources/icon.ico",
+      "installerIcon": "resources/icon.ico",
+      "uninstallerIcon": "resources/icon.ico",
+      "installerSidebar": "resources/installerSidebar.bmp",
+      "uninstallerSidebar": "resources/installerSidebar.bmp",
+      "allowToChangeInstallationDirectory": true,
+      "license": "resources/eula.txt"
+    },
+    "dmg": {
+      "icon": "resources/icon.icns",
+      "iconSize": 128,
+      "contents": [
+        {
+          "x": 380,
+          "y": 240,
+          "type": "link",
+          "path": "/Applications"
+        },
+        {
+          "x": 122,
+          "y": 240,
+          "type": "file"
+        }
+      ]
+    },
+    "afterSign": "scripts/notarize.js",
+    "publish": [
+      {
+        "provider": "s3",
+        "bucket": "arduino-downloads-prod-beagle",
+        "path": "arduino-ide/nightly"
+      }
+    ]
+  },
+  "theiaPluginsDir": "plugins",
+  "theiaPlugins": {
+    "vscode-builtin-cpp": "https://open-vsx.org/api/vscode/cpp/1.52.1/file/vscode.cpp-1.52.1.vsix",
+    "vscode-arduino-api": "https://github.com/dankeboy36/vscode-arduino-api/releases/download/0.1.2/vscode-arduino-api-0.1.2.vsix",
+    "vscode-arduino-tools": "https://downloads.arduino.cc/vscode-arduino-tools/vscode-arduino-tools-0.0.2-beta.8.vsix",
+    "vscode-builtin-json": "https://open-vsx.org/api/vscode/json/1.46.1/file/vscode.json-1.46.1.vsix",
+    "vscode-builtin-json-language-features": "https://open-vsx.org/api/vscode/json-language-features/1.46.1/file/vscode.json-language-features-1.46.1.vsix",
+    "cortex-debug": "https://downloads.arduino.cc/marus25.cortex-debug/marus25.cortex-debug-1.5.1.vsix",
+    "vscode-language-pack-bg": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-bg/1.48.3/file/MS-CEINTL.vscode-language-pack-bg-1.48.3.vsix",
```

Comment:
update language VSIX

Reply:
The baseline should be: https://github.com/eclipse-theia/theia/blob/v1.39.0/dev-packages/application-package/src/api.ts#L21

Reply:
https://github.com/arduino/arduino-ide/commit/661fe1d8cadfa42d81fb71aaee55f6079ba60c5a